certified-connectors/DocuSignDemo/script.csx

```diff
@@ -1639,21 +1639,27 @@ public class Script : ScriptBase
     {
       var body = ParseContentAsJObject(await response.Content.ReadAsStringAsync().ConfigureAwait(false), false);
       var query = HttpUtility.ParseQueryString(this.Context.Request.RequestUri.Query);
-      var tabType = query.Get("tabLabel");
+      var tabLabel = query.Get("tabLabel");
       var newBody = new JObject();
 
       foreach(JProperty tabTypes in body.Properties())
       {
         foreach(var tab in tabTypes.Value)
         {
-          if((tab["tabLabel"].ToString()).Equals(tabType.ToString()))
+          if((tab["tabLabel"].ToString()).Equals(tabLabel.ToString()))
```

Comment:
For the above mentioned reason, a ToString() on a String type is not needed. Just change to .Equals(tabLabel) and it should work. Please test if so.

Reply:
You are correct. String conversion is not required here. I have tested  it and it is working.